Mathlib/LinearAlgebra/FreeModule/StrongRankCondition.lean

```diff
@@ -19,46 +19,43 @@ module is well defined.
 
 ## Main result
 
+* `commRing_orzechProperty R` : `R` has the `OrzechProperty`.
```

Comment:
Can you add the statement of the Orzech property in the docstring?

Reply:
Done.